tests/schemas/random.yaml

```diff
@@ -0,0 +1,422 @@
+---
+version: v1-draft
+exports:
+  noInputExport:
```

Comment:
I had Claude generate some ruby code which generates this document. It's not perfect, but it helped smoke out a couple problems. I want to explore writing another program that will generate a valid document that exercises every case.